[2022-11-27T09:55:32.078+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2022-11-27T09:55:21.105109+00:00 [queued]>
[2022-11-27T09:55:32.149+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2022-11-27T09:55:21.105109+00:00 [queued]>
[2022-11-27T09:55:32.153+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-27T09:55:32.156+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-27T09:55:32.159+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-27T09:55:32.211+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2022-11-27 09:55:21.105109+00:00
[2022-11-27T09:55:32.242+0000] {standard_task_runner.py:55} INFO - Started process 5197 to run task
[2022-11-27T09:55:32.257+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2022-11-27T09:55:21.105109+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpi5e5vram']
[2022-11-27T09:55:32.274+0000] {standard_task_runner.py:83} INFO - Job 30: Subtask is_api_available
[2022-11-27T09:55:32.525+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2022-11-27T09:55:21.105109+00:00 [running]> on host c7f7231a8603
[2022-11-27T09:55:32.800+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2022-11-27T09:55:21.105109+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-27T09:55:21.105109+00:00
[2022-11-27T09:55:32.804+0000] {http.py:120} INFO - Poking: api/
[2022-11-27T09:55:32.852+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2022-11-27T09:55:32.865+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2022-11-27T09:55:33.393+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2022-11-27T09:55:33.522+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20221127T095521, start_date=20221127T095532, end_date=20221127T095533
[2022-11-27T09:55:33.749+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-27T09:55:34.006+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
